### ISSUE TITLE
Can I get the original interpolated data but not Isosurface？

### ISSUE THREAD

**Issue by raypenper** [polatory/polatory#46]
@mizuno-gsinet I am sorry to borther you with another issue. but i extremely eager to get the whole data after interpolation. I believe the function `resolution` is what i want. when i debugged it. i got only a vector which has the same size as original data.

![image](https://user-images.githubusercontent.com/41855946/54484553-ec218680-48a3-11e9-93c4-acd5419cbffa.png)

**Comment by mizuno-gsinet** [comment]
@raypenper There are two stages in RBF interpolation:

1. Fitting: to solve a system of linear equations to obtain the weights of basis functions
2. Evaluation: to evaluate the weighted sum of basis functions (=interpolant) on a set of points

Once the fitting is done, you can evaluate the interpolant on as many points as you want.

According to your previous edits, I believe you are trying to evaluate the interpolant on a grid of points, right? I have modified kriging_predict for the purpose. Read through and give it a try.

https://gist.github.com/mizuno-gsinet/779ccb554c20a7894b895c5afb179998

**Comment by raypenper** [comment]
@mizuno-gsinet Thanks a lot, that's exactly what i want. i will give it a try.
